src/js/hooks/useDeleteNotificationListener.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+
+import _ from 'lodash';
+import queryString from 'query-string';
+import { useHistory, useLocation } from 'react-router-dom';
+
+import notification from 'components/Layout/notifications/notification';
+import NotificationType from 'consts/notificationTypes';
+
+const useDeleteNotificationListener = () => {
```

Comment:
`useDeleteNotificationListener` seems a bit too specific, shouldn't it be more generic like `useQueryMessageNotificationListener` ?